components/FriendPagePreview.js

```diff
@@ -70,33 +88,62 @@ export default function FriendPagePreview({ friend }) {
           height={180}
         />
       </div>
-
       {/* <p>Contact:</p> */}
+
       <StyledContactList>
-        {Object.entries(friend.contactOptions).map(([key, value]) => (
-          <StyledList key={key}>
-            <StyledButton href={value}>
-              {key}{" "}
-              {/* : <br />
+        {friend.contactOptions &&
+          friend.contactOptions.map((options) => (
+            <StyledList key={options.name}>
```

Comment:
The naming of `StyledList`is quite confusing. `StyledList` would suggest that this styled component is based on a `ul` or `ol`, but it`s a `li`. The name then should be `StyledListItem`